plugins/terminal/eos.py

```diff
@@ -52,6 +52,9 @@ class TerminalModule(TerminalBase):
         re.compile(br"% Subnet [0-9a-f.:/]+ overlaps", re.I),
         re.compile(br"Maximum number of pending sessions has been reached"),
         re.compile(br"% Prefix length must be less than"),
+        re.compile(
+            br"% Cannot change mode; remove all members and try again.", re.I
```

Comment:
Is `re.I` necessary here?

Also, can you please add a comment noting what sort of command causes this response?

Reply:
I added re.I as a precaution, if there is a change in the case in future. But now that I think about it, the chance for such change is slim.